File: C++ programming/PROBLEMS ONC NUMBER/Program1.cpp
#include<iostream>
using namespace std;

bool CheckFactor(int iNo1 , int iNo2)
{
    if((iNo1 % iNo2) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }
}
int main()
{ 
  int No1 = 0;
  int No2= 0;
  bool bRet = false;

  cout<<"Enter First Number : ";
    cin>>No1;

    cout<<"Enter Second  Number :";
    cin>>No2;

    bRet = CheckFactor(No1 , No2);
    if(bRet == true)
    {
        cout<<No2<<" is a factor of "<<No1 ;
    }
    else
{
        cout<<No2<<" is a not factor of "<<No1 ;

}
    return 0;
}

File: JAVA Programming/PROBLEMS ON NUMBER/Program1.java
import java.util.*;
class Program1
{
    public static boolean CheckFactor(int iNo1 , int iNo2)
    {
        if((iNo1 % iNo2) == 0)
        {
            return true;
        }
        else
            return false;
    }
    public static void main(String Args[])
    {
        boolean bRet = false;
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter First Number :");
        int iNo1 = sobj.nextInt();
        System.out.println("Enter Second Number :");
        int iNo2 = sobj.nextInt();

        bRet = CheckFactor(iNo1 , iNo2);
        if(bRet == true)
        {
            System.out.println(iNo2 + " Number is Factor of " + iNo1);
        }
        else
            System.out.println(iNo2 + " Number is not  Factor of " + iNo1);
        {
        }


    }
}